server/app.js

```diff
@@ -0,0 +1,66 @@
+    
+if(process.env.NODE_ENV !== 'production'){
+  require('dotenv').config()
+} 
+//DEPENDECIES
+const path = require('path')
+const http = require('http');  
+const createError = require('http-errors');
+const express = require('express');
+const app = express();
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
+const cors = require('cors');
+const passport = require ('passport');
+/* const socketio = require('socket.io')
+ */
+//Routes
+const registerRouter = require('./routes/register');
+const loginRouter = require('./routes/login');
+const dashboardRouter = require('./routes/dashboard');
+const authrouter = require('./routes/authentication')
+
+//configurations
+const db = require("./database/configurationSequelize")
+require('./Configurations/helper/passportConfig')(passport)
+db.connector.sync();
+require('./Configurations/googleAuth/passportGoogleConfig')(passport)
+
+/* const server = http.createServer(certOptions, app)
```

Comment:
Consider removing comments of things that aren't directly being used in your file. Line 29 for ```server``` and lines 49-52 for ```socketio```